motioneye/static/css/main.css

```
@@ -1026,24 +1027,74 @@ div.mask-element.on {
     background: rgba(255, 0, 0, 0.5);
 }
 
-div.camera-name {
-    display: inline-block;
-    font-size: 1.2em;
-    font-weight: bold;
+div.camera-info {
+    display: flex;
+    font-size: 0.8em;
     white-space: nowrap;
     width: 33.33%;
     overflow: hidden;
     text-overflow: ellipsis;
     vertical-align: top;
+    text-align: left;
+    line-height: 1em;
+    flex-grow: 2;
+    flex-direction: column;
+}
+
+div.camera-info-top {
+    display: flex;
+    flex-grow: 2;  
```

Remove trailing space:
```suggestion
    flex-grow: 2;
```